src/purify.ts

```diff
@@ -30,11 +41,16 @@ export function purify(content: string) {
     )
     /* Prefix CCF and CMF statements */
     .replace(
-      /__WEBPACK_IMPORTED_MODULE_\d+__angular_core__\["_?\w+" \/\* (ɵccf|ɵcmf) \*\/\]\(/mg,
+      /\w*__WEBPACK_IMPORTED_MODULE_\d+__angular_core__\["\w+" \/\* (ɵccf|ɵcmf) \*\/\]\(/mg,
+      '/*@__PURE__*/$&',
+    )
+    /* Prefix module statements */
+    .replace(
+      /new \w*__WEBPACK_IMPORTED_MODULE_\d+__angular_core__\["\w+" \/\* NgModuleFactory \*\/\]/mg,
       '/*@__PURE__*/$&',
     )
     .replace(
-      /\/\*\*\r?\n\s*\*\s?@license.*\r?\n(\s*\*[^\/].*\r?\n)*\s*\*\//mg,
+      /\/\*\*\r?\n\s*\*\s?@license.*\r?\n((\s*\*[^\/]|\/\*\*).*\r?\n)*\s*\*\//mg,
```

Comment:
we don't need this feature any more since it was a workaround for a bug in the core that already got fixed. Can you please remove this?